### DISCUSSION TITLE
Group Instance Feature

### DISCUSSION THREAD

**Discussion by AariyJP** [vrcm-team/VRCM#2] [Ideas]
Thank you for developing this useful app.

I wish it could show group instances as well.

I'm thinking of using it as an alternative to vrchat.com/home, but I can't see group instances in this app like I can on the site.

If you don't mind, I'd appreciate it if you could make one.

**Comment by KAMO030** [comment]
ご利用ありがとうございます
group instancesの表せ方はどうな方でいいですか？
FriendLocal instancesと同じ一緒に表すか それともウェブの様に別のタイトルで分ける方が良いですか？
それで いまのFriendLocal instancesとの順番を考えるべきです
良ければ ご意見を参考させていただいたらうれしいです

**Reply by mdogwoop** [reply]
可以参考一下官方app的实现，一个群组里有简介，公告等页面，用friend页面的话功能好像就有点太少了
![Screenshot_20260616-163538.png](https://github.com/user-attachments/assets/55ca398f-cb9a-48a5-8e3a-345cdfdf3a2f)

**Comment by yourmoln** [comment]
This feature is coming in the next update! Check out the UI preview below.
<img width="874" height="1920" alt="6a5a596cba09bcd10c8a822b7043ffc4_720" src="https://github.com/user-attachments/assets/07b2bbe6-0e66-4749-a6bd-a0f8682f1890" />
